gateway-sp-comms/src/single_sp.rs

```diff
@@ -811,6 +811,25 @@ impl SingleSp {
                 response.expect_component_action_ack()
             })
     }
+
+    pub async fn read_component_caboose(
```

Comment:
We probably don't want to remove `SingleSp::get_caboose_value()` (the old getter) immediately, but it's maybe worth a comment that we should remove it at some point once all the SPs we care about have been updated. Similarly, there's one use of a raw `MgsRequest::ReadCaboose` in `single_sp/update.rs`; we can't change that one yet either (because it prevent updating SPs that don't know about per-component caboose messages), but it could also use a comment that it should use the new request variant at some point?

Reply:
Good point, added some comments in 6834e9d3b91b591d6d061e090276731354699730